src/components/2023/nav.js

```diff
@@ -5,33 +5,33 @@ import { homeBackground } from "./cloudImages";
 
 const routes = [
   {
-    route: "/2023/#about",
+    route: "/#about",
     title: "About"
   },
   {
-    route: "/2023/#speakers",
+    route: "/#speakers",
     title: "Speakers"
   },
   {
-    route: "/2023/schedule",
+    route: "/schedule",
     title: "Schedule"
   },
   {
-    route: "/2023/sustain-africa",
+    route: "/sustain-africa",
     title: "Sustain Africa"
   },
   {
-    route: "/2023/travel",
+    route: "/travel",
     title: "Travel"
   },
   {
     route: "https://www.flickr.com/photos/oscafrica/albums/with/72177720298698442",
```

Comment:
Note for another PR: Ideally, the gallery here should link to a dedicated gallery page like [this](https://festival.oscafrica.org/2022/gallery) (the same component we already have on the homepage, just that we'd need that path for further custom social link sharing).